src/components/AddBook.js

```diff
@@ -53,7 +53,7 @@ const AddBook = () => {
           required
         />
         <select>
-          <option>Category</option>
+          <option className="default-option">Category</option>
           <option value="option1">Action</option>
           <option value="option2">fiction</option>
           <option value="option3">Romance</option>
```

Comment:
- [ ] As the previous reviewer suggested, please fix layout of the form so that the "Add Book" button is moved toward the far-right (end) of its container as shown in the guideline below 👇 
## Your Design
![image](https://github.com/hamzaw9/book-store/assets/46839548/093809cb-2b22-4ecd-82d7-75088a82d34f)
## Zeplin Design
![image](https://user-images.githubusercontent.com/81727988/282230803-a7c4a050-84fe-4e5f-8918-fdb016e50ca7.jpg)

Reply:
Kindly see the attached screenshot. You can see on my browser the button is placed where it's needed to be. 
![Screenshot](https://github.com/hamzaw9/book-store/assets/92848670/315b0d58-6382-4d5d-b6ba-091731955917)